File: Java/Important-Programs/10-sum-of-digits-of-a-number.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter a number:");
        int num = getInput.nextInt();
        int sum = 0;

        while (num != 0) {
            int digit = num%10;
            sum += digit;
            num /= 10;
        }

        System.out.println("Sum is: " + sum);
        getInput.close();
    }
}

File: Python/Important-Programs/10-sum-of-digits-of-a-number.py
num = int(input("Enter a number: "))
sum = 0

while num!=0:
    digit = int(num%10)
    sum += digit
    num = num//10

print("Sum is:", sum)

# Using recursion
def findSum(num):
    if num == 0:
        return sum
    return int(num%10) + findSum(num//10)

print("Sum is:", findSum(num))